python/ray/autoscaler/_private/kuberay/autoscaling_config.py

```diff
@@ -333,6 +358,31 @@ def _get_num_gpus(
     return None
 
 
+def _get_num_tpus(
```

Comment:
It would be useful to have a unit test for this function for reviewers (like me 😅) who aren't sure what `ray_start_params`, `k8s_resource_limits`, `group_name` are typically.

Reply:
Done.